spec/javascripts/inline_grids_spec.js

```diff
@@ -1,39 +1,31 @@
-//= require application
-//= require jasmine-jquery
-//= require comment_expand
-
 var editor;
 
+fixture.preload("inline_grid.html");
+
 describe("Inline grids", function() {
 
   beforeEach(function() {
 
-    // for phantomjs running
-    jasmine.getFixtures().fixturesPath="../../spec/javascripts/fixtures";
-
-    // for in-browser running... still doesn't work
-    //jasmine.getFixtures().fixturesPath = 'assets/fixtures';
-//    preloadFixtures('inline_grids.html');
+      this.fixtures = fixture.load("inline_grid.html");
 
   });
 
 
   it("sends a like request when like button is clicked", function() {
 
-    loadFixtures('inline_grid.html');
 
-    expect($($('.notes-grid tr')[1]).find('td:first a').html()).toEqual('First post');
+    expect($($('.notes-grid tr')[1]).find('td:first a').html()).to.eql('First post');
 
     $('table:first th:first a').trigger('click');
 
     setTimeout(0, function() {
```

Comment:
Hi @jywarren, can I know why we used setTimeout here. Can't we test this without it?